vendor-extra/JatsContentBundle/src/ViewConverter/HeadingVisitor.php

```diff
@@ -31,11 +31,6 @@ protected function expectedTemplate() : string
         return '@LiberoPatterns/heading.html.twig';
     }
 
-    protected function expectedElement() : string
-    {
-        return '{http://jats.nlm.nih.gov}article-title';
```

Comment:
Not sure what's the best approach to also add in support for `{http://jats.nlm.nih.gov}title`. Can see one of

- Separate visitor (explicit, more code that does exactly the same)
- Allow multiple possible elements here (explicit and reuses same code)
- Don't expect a specific element, like this (might behave strangely if given an element that isn't title like); means no code changes to support `{http://jats.nlm.nih.gov}trans-title` etc

Reply:
Configurable set of elements for a single Visitor?